docs/standard/components.md

```diff
@@ -0,0 +1,80 @@
+---
+title: .NET Concepts
+description: Describes key .NET concepts such as the .NET Standard Library, .NET runtimes, tooling, and how they interplay.
+keywords: .NET, .NET Standard Library, .NET Standard, .NET Core, .NET Framework, Xamarin, MSBuild, C#, F#, VB, compilers
+author: cartermp
+manager: wpickett
```

Comment:
we're also adding ms.author which is the person who is going to be responsible for maintain and receive feedback bugs, BI data, etc.